src/property-filter/index.tsx

```diff
@@ -361,7 +367,7 @@ const PropertyFilter = React.forwardRef(
             ariaLabel={filteringAriaLabel ?? i18nStrings.filteringAriaLabel}
             placeholder={filteringPlaceholder ?? i18nStrings.filteringPlaceholder}
             ariaLabelledby={rest.ariaLabelledby}
-            ariaDescribedby={rest.ariaDescribedby}
+            ariaDescribedby={textboxAriaDescribedBy}
```

Comment:
now we just need a test for this and we should be good :)

Reply:
🤦‍♂️ My bad, that's what I get for rushing the second revision.

Reply:
Done. Sorry for the extra revisions on this one, I got too comfortable. 😛